provider/README.md

```diff
@@ -302,6 +303,22 @@ When multiple query parameters are specified, they should all apply to the retur
 | | | `rebalance_pick_up` | Device removed from street and will be placed at another location to rebalance service |
 | | | `maintenance_pick_up` | Device removed from street so it can be worked on |
 
+[Top][toc]
+
+## Versions
+
+The versions endpoint allows user to query for the versions the provider supports.
+
+Endpoint: `/versions`  
+Method: `GET`  
+Schema: [`versions` schema][versions-schema]  
+`data` Payload: `{ "versions": [] }`, an array of version strings usable in versioned requests
+
+Here is an example payload:
+
+    "versions": [ "0.2", "0.3" ]
```

Comment:
it's probably worth documenting somewhere about the rationale of trimming these to minor version only.

Reply:
I've updated this do the negotiation using the `OPTIONS` method and am now returning the whole string a client would use in the request `ACCEPT` header. It is now made explicit in the README.md file.